### PR TITLE
Fd01 first onboarding screen

### PR DESCRIPTION
**Pull request by mervenurgulbagci**
add first onboarding screen

### CODE REVIEW

File: Foody/Onboarding/FirstScreen/FirstViewController.swift [hunk 1]
```diff
@@ -10,46 +10,32 @@ import UIKit
...
-    @objc func nextButtonClicked(){
+    
+    @IBAction func nextButtonClicked(_ sender: Any) {
         nextButtonCompletion?()
     }
+    
+    
+    func setupView(){
+        appNameLbl.text = "FOODY"
+//        view.addSubview(appNameLbl)
```
Comment (gokcesrslmz) [inline Foody/Onboarding/FirstScreen/FirstViewController.swift]: Please remove comment out code (// view.addSubview(appNameLbl))

File: Foody/Onboarding/FirstScreen/FirstViewController.swift [hunk 2]
```diff
@@ -10,46 +10,32 @@ import UIKit
...
+        descriptionLbl.text = "Discover Delicious Recipes"
+//        view.addSubview(descriptionLbl)
```
Comment (gokcesrslmz) [inline Foody/Onboarding/FirstScreen/FirstViewController.swift]: Please remove comment out code (// view.addSubview(appNameLbl))

File: Foody/Onboarding/FirstScreen/FirstViewController.swift [hunk 3]
```diff
@@ -10,46 +10,32 @@ import UIKit
...
+        descLbl.text = "Easily find the best recipe you are looking for with all the details"
+//        view.addSubview(descLbl)
```
Comment (gokcesrslmz) [inline Foody/Onboarding/FirstScreen/FirstViewController.swift]: Please remove comment out code (// view.addSubview(appNameLbl))

**Review by gokcesrslmz — CHANGES_REQUESTED** [review]

**Comment by gokcesrslmz** [comment]
Simulator Screen Shot - iPod touch (7th generation) - 2022-12-28 at 18 57 57
@mervenurgulbagci  ortadaki text guzel gozukmuyor. Labelin line countunu 0 yapabilrsin. Bunun disinda saga ve sola cok yanasik duruyorlar. Stackviewim sagina ve soluna belki biraz daha bosluk verebilirsin.

File: Foody/Onboarding/FirstScreen/FirstViewController.swift (hunk 1 above)
Comment (mervenurgulbagci) [reply Foody/Onboarding/FirstScreen/FirstViewController.swift]: Got it.

File: Foody/Onboarding/FirstScreen/FirstViewController.swift (hunk 2 above)
Comment (mervenurgulbagci) [reply Foody/Onboarding/FirstScreen/FirstViewController.swift]: Got it.

File: Foody/Onboarding/FirstScreen/FirstViewController.swift (hunk 3 above)
Comment (mervenurgulbagci) [reply Foody/Onboarding/FirstScreen/FirstViewController.swift]: Got it.

**Pull request closed without merging**